### PR TITLE
Devrel rework - add example notebook

### PR DESCRIPTION
**Pull request by andrewhong5297**

### CODE REVIEW

File: examples/Protocol Wallet Clustering - Analyzing all Artblocks Collectors.ipynb
[the remark this answers is not in the document]
Comment (bh2smith) [reply examples/Protocol Wallet Clustering - Analyzing all Artblocks Collectors.ipynb:1]: +1

File: README.md [hunk 1]
```diff
@@ -57,10 +57,12 @@ Use `get_latest_results` to get the most recent query results without using exec
 You can specify a `max_age_hours` to re-run the query if the data is too outdated.
 
 ```python
+import pandas as pd
 from dune_client.client import DuneClient
 
 dune = DuneClient.from_env()
 results = dune.get_latest_result(1215383, max_age_hours=8)
```
Comment (msf) [inline README.md:64]:
```suggestion
query_results = dune.get_latest_result_dataframe(1215383)
```

**Review by msf — APPROVED** [review]
can you update the jupyter notebook to use `get_latest_result_dataframe()` where appropriate.
 :-) 

thanks, approving it so you don't have to wait for another review

File: README.md (hunk 1 above)
Comment (bh2smith) [reply README.md:64]: This suggested change breaks the context of the readme. Specifically demonstrating the use of max age.

Comment (bh2smith) [reply README.md:64]: That's fine, but then the readme should probably be changed to remove the max age example.

**Comment by msf** [comment]
closing this PR, it is stale by now

**Pull request closed without merging**